Dockerfile

```diff
@@ -12,7 +12,7 @@ ENV \
 ARG QEMU_CPU
 
 # Install uv
-RUN pip3 install uv==0.4.28
+RUN pip3 install --no-cache-dir uv==0.4.28
```

Comment:
This is already handled upstream: 

https://github.com/home-assistant/docker-base/blob/26a71a045dcd0f2622ffdc0200426c97a31e3f2e/alpine/rootfs/etc/pip.conf#L4